lib/Frontend/CompilerInvocation.cpp

```diff
@@ -1273,6 +1275,12 @@ static bool ParseClangImporterArgs(ClangImporterOptions &Opts,
 
   Opts.DumpClangDiagnostics |= Args.hasArg(OPT_dump_clang_diagnostics);
 
+  if (FrontendOpts.RequestedAction != FrontendOptions::ActionType::REPL &&
+      FrontendOpts.RequestedAction != FrontendOptions::ActionType::NoneAction &&
```

Comment:
When invoked directly (ie. `lldb --repl="...") the `RequestedAction` seems to be `NoneAction`.

Reply:
Should this be a comment in the code?